pkg/leech/ingest_logs.go

```diff
@@ -150,12 +153,15 @@ func (f *IngestLogsFn) ProcessElement(ctx context.Context, event EventRecord) Le
 		RepositoryName:   event.RepositoryName,
 		RepositorySlug:   event.RepositorySlug,
 		LogsURI:          gcsPath,
-		Status:           "SUCCESS",
+		Status:           "SUCCESS", // maybe initialize as PENDING and then update in/after handleMessage?
```

Comment:
In the majority of cases this should be `SUCCESS` but you certainly could do something else here.